path_role.go

```diff
@@ -69,6 +69,21 @@ be renewed. Defaults to 0, in which case the value will fall back to the system/
 should never expire. The token should be renewed within the
 duration specified by this value. At each renewal, the token's
 TTL will be set to the value of this parameter.`,
+			},
+			"expiration_leeway": {
+				Type: framework.TypeDurationSecond,
+				Description: `Duration in seconds of leeway when validating expiration of a token to account for clock skew. 
+Defaults to 150 (2.5 minutes).`,
+			},
+			"not_before_leeway": {
+				Type: framework.TypeDurationSecond,
+				Description: `Duration in seconds of leeway when validating not before values of a token to account for clock skew. 
+Defaults to 150 (2.5 minutes).`,
+			},
+			"clock_skew_leeway": {
```

Comment:
Why a bool instead of a duration? Based on past issues and feedback different users have different clock skew tolerances.

Reply:
I can make this a duration if you want but I thought we just wanted to toggle the JWT package default on and off based on your last review:

> ... where we default to the library's default but people can turn it _off if they don't want an extra minute of validity (which they may well not want)._

Reply:
Thanks for changing this. I had simply meant using its value as the default. Note that you can(/should) put a `Default` for each of these, which then allows that to show up in OpenAPI. For the clock skew leeway the value could just be `jwt.DefaultLeeway` or whatever it's called.